src/core/messaging/types.ts

```diff
@@ -1,9 +1,11 @@
+// TODO: refactor this, too many types
 export enum NotificationType {
     EXTENSION_TRANSACTION = 'EXTENSION_TRANSACTION',
```

Comment:
We need to cleanup this enum ,,, what's the difference between EXTENSION_TRANSACTION and EXTENSION_SIGN_TX or MOONLET_TRANSFER